torch_xla/csrc/ops/einsum.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "torch_xla/csrc/ir.h"
+
+namespace torch_xla {
+
+class Einsum : public XlaNode {
+ public:
+  Einsum(const torch::lazy::OpList& operands, const std::string equation);
+
+  torch::lazy::NodePtr Clone(torch::lazy::OpList operands) const override;
+
+  XlaOpVector Lower(LoweringContext* loctx) const override;
+
+  std::string ToString() const override;
+
+  const std::string& equation() const { return equation_; }
+
+ private:
+  const std::string equation_;
```

Comment:
hmm, curious what does a const private variable means? It can not be modified after init?

Reply:
Yes, and you need to initialize the variable in an initializer list rather than the ctor body

https://stackoverflow.com/questions/16167649/the-constructor-initializer-list-and-const-variable